data/mods/blazemod/items/vehicle/blob_magazines.json

```diff
@@ -0,0 +1,20 @@
+[
```

Comment:
Simply separated out, notable for being a `"type": "MAGAZINE"` while in the `veh_parts` category.